src/components/Login.vue

```diff
@@ -39,24 +65,54 @@ import firebase from "firebase";
 export default class Login extends Vue {
   private email = "";
   private password = "";
+  public errorMessage = "";
+  public showAlert = false;
+  public showPassword = false;
 
-  public async logToFirebase() {
-    await firebase.auth().signInWithEmailAndPassword(this.email, this.password);
+  public async logToFirebase(event: any) {
+    event.preventDefault();
+    await firebase
+      .auth()
+      .signInWithEmailAndPassword(this.email, this.password)
+      .then(() => {
+        this.$bvModal.hide("modal-login");
+      })
+      .catch((error) => {
+        switch (error.code) {
+          case "auth/invalid-email":
+            this.errorMessage = "EMAIL_INVALID";
+            break;
+          case "auth/user-not-found":
+            this.errorMessage = "EMAIL_NOT_FOUND";
+            break;
+          case "auth/user-disabled":
+            this.errorMessage = "USER_DISABLED";
+            break;
+          case "auth/wrong-password":
```

Comment:
en terme de sécu c'est mieux de mettre le même message d'erreur pour invalid-email user-not-found et wrong-password